core/web/__init__.py

```diff
@@ -0,0 +1,121 @@
+from flask import Flask, cli, render_template
+from flasgger import Swagger
+from core import initial
+from core import core
+from redis import Redis
+import rq
+import os
+import requests
```

Comment:
Can't you use the request function?
We have to use the set framework options.

Reply:
I have not used the code from recon-ng but as the previous version of this project was heavily inspired by recon-ng I just used the same web stack such as flask, swagger, and Redis. and as there is a standard way to initialize the modules used so init.py file will look similar. other than that the HTML, CSS, JS and API code are all  that I have written without any inspiration.

Reply:
Aman, for me, there is no problem with inspiration from another project. For OWASP Project Committee. It doesn't make sense and because of that, I rewrote the core.
You can continue your work.